### DISCUSSION TITLE
Takes too long to generate window post

### DISCUSSION THREAD

**Discussion by wyfor4667** [filecoin-project/lotus#8508] [General Q&A] [answered]
I have about 8 deadlines.
And all window posts of each deadline were used to proceeded well always.
However after some crash happened on window post process at deadline 4 as linked below,
generate_window_post time takes too long only at deadline 4.

https://filecoinproject.slack.com/archives/CPFTWMY7N/p1649011439657799

There is no specific error without error code 19 and context deadline exceeded on lotus (lotus log link attached below)

https://filecoinproject.slack.com/files/U02453DSBLJ/F03BZUPJ9S7/lotus.log

I tried many things
There are basic things I want to share the facts might be useful to help

proving check --slow deadline 4 takes less than 20min always and there is no any bad sector.
sometimes deadline 4 passed window post at time,  however normally the elapsed time is over 3000s, resulting in faults.

io(input/output interface) speed is quite slow only at proving check through NFS.
but in 'dd' read command is 40 times faster (~400MB/s with 10Gb NIC NFS server roll only system). 

I severly have got stressed out because of the penalty policy of filecoin
Plz help me

**Comment by LexLuthr** [comment, marked as answer]
Please use the new windowPost workers and check if the situation improves.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
